### DISCUSSION TITLE
Removing Artists and Tracks from 'Home'

### DISCUSSION THREAD

**Discussion by chongwho** [music-assistant/support#3211] [Q&A] [answered]
Hi team,

Am I able to remove or at least re-order the categories on the front page? 

Apologies if this has already been asked. I'm a HA/Github noob.

![image](https://github.com/user-attachments/assets/922e548f-b0f0-48b5-9b63-a5caa816d392)

**Comment by OzGav** [comment]
No sorry. You can only change the menu items at the bottom

**Reply by chongwho** [reply]
Ok thank you

**Reply by OzGav** [reply]
UPDATE: The home screen is now fully configurable

**Comment by OzGav** [comment, marked as answer]
This has now been implemented in 2.5.0b18

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
